Screen-Sharing/Screen-Sharing/ScreenCapturer.swift

```diff
@@ -81,6 +88,7 @@ class ScreenCapturer: NSObject, OTVideoCapture {
     
     // MARK: - OTVideoCapture protocol
     func initCapture() {
+        timerState = .unknown
```

Comment:
Should `unknown` be `notStarted` or `off` or  something like that? When I think of timers it is on or off or paused or  notStarted.

Reply:
I have included `notStarted` and other timer states. Please let me know if it's OK for you!